### DISCUSSION TITLE
livewire full page compoment embedded in  iframe

### DISCUSSION THREAD

**Discussion by aftabkhaliq86** [livewire/livewire#7563] [1. Help]
I need help embedding the iframe of my subdomain in another Laravel site.
the issue is with the csrf token mismatch 
I don't want to put  'livewire/update' middleware except as I want to add specific   routes of livewire in it

**Comment by PhiloNL** [comment]
> I need help embedding the iframe of my subdomain in another Laravel site.
> 
> the issue is with the csrf token mismatch 
> 
> I don't want to put  'livewire/update' middleware except as I want to add specific   routes of livewire in it 

You might want to disable it as iframes don't play nice anymore with cookies when the iframe is loaded from a different domain.

In v2 I did something like:

```
if (request()->is('livewire/message/widget')) {
    config(['livewire.middleware_group' => 'api']);
}
```

In v3 you could do something similar and inspect the request payload and disable the csrf if it matches your component.

**Reply by aftabkhaliq86** [reply]
is there any way that i can get component name  of the request ?
because I don't want to disable it on all components livewire/update

**Reply by alexjustesen** [reply]
Following along, hurdle I'll need to solve for on https://github.com/alexjustesen/speedtest-tracker for embedding into home lab dashboards.

**Reply by bilogic** [reply]
@PhiloNL any examples on how to do this in v3? thanks

**Reply by PhiloNL** [reply]
I ended up with something like this:
https://github.com/wire-elements/wire-extender/blob/main/src/Http/Middlewares/IgnoreForWireExtender.php

**Reply by bilogic** [reply]
Thanks! Within the short 11 minutes, I managed to find https://stackoverflow.com/a/69767153/3872647 that overcame the problem by setting

```
'same_site' => 'none',
SESSION_SECURE_COOKIE=true
```

As I'm new to the problem, would you be able to explain the pros/cons of both solutions?
